src/utils/fetchTotalValidators.ts

```diff
@@ -0,0 +1,36 @@
+import axios, { AxiosResponse } from 'axios';
+import { BEACONCHAIN_URL } from './envVars';
+
+interface EpochData {
+  data: {
+    validatorscount: number;
+  };
+}
+export interface FetchTotalValidatorsResponse {
+  statusCode: number;
+  body: {
+    totalValidators: number;
+    msg?: unknown;
+  };
+}
+
+export const fetchTotalValidators = async (): Promise<FetchTotalValidatorsResponse> => {
+  const { href } = new URL('/api/v1/epoch/latest', BEACONCHAIN_URL);
+  try {
+    const response: AxiosResponse = await axios.get(href);
+    if (response.status !== 200) throw new Error(response.statusText);
+    const { data }: { data: EpochData } = response;
```

Comment:
same here with the types around axios

Reply:
Nice, thanks for suggesting this...  much cleaner